### PR TITLE
[BEAM-14078] [Website] change link

### PR DESCRIPTION
**Pull request by bullet03**
[BEAM-14078]
- update link:
from (https://frontend-beta-dot-apache-beam-testing.appspot.com/)
to (https://play.beam.apache.org/)

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by bullet03** [comment]
R: @AydarZaynutdinov

**Review by AydarZaynutdinov — APPROVED** [review]
LGTM

File: website/www/site/content/en/get-started/try-beam-playground.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Beam Playground is an interactive environment to try out Beam transforms and exa
 without having to install Apache Beam in your environment.
 
 You can try an Apache Beam examples at
```
Comment (AydarZaynutdinov) [inline website/www/site/content/en/get-started/try-beam-playground.md:23]:
```suggestion
You can try the available Apache Beam examples at
```

**Review by AydarZaynutdinov — CHANGES_REQUESTED** [review]

**Comment by dariabezkorovaina** [comment]
@pcoet please take a look:)

File: website/www/site/content/en/get-started/try-beam-playground.md [hunk 2]
```diff
@@ -20,8 +20,8 @@ limitations under the License.
 Beam Playground is an interactive environment to try out Beam transforms and examples
 without having to install Apache Beam in your environment.
 
-You can try an Apache Beam examples at
-[Beam Playground](https://frontend-beta-dot-apache-beam-testing.appspot.com/).
+You can try the available Apache Beam examples a
```
Comment (pcoet) [inline website/www/site/content/en/get-started/try-beam-playground.md:23]: This should be "examples at", right?

**Review by pcoet — CHANGES_REQUESTED** [review]

File: website/www/site/content/en/get-started/try-beam-playground.md (hunk 2 above)
Comment (AydarZaynutdinov) [reply website/www/site/content/en/get-started/try-beam-playground.md]: Changed.

**Review by pcoet — APPROVED** [review]

**Comment by pabloem** [comment]
LGTM thanks!

**Pull request merged**